app/views/posts/show.html.erb

```diff
@@ -1,2 +1,22 @@
-<h1> User posts page </h1>
-<p> Details of a specific post of a user
+<section>
+  <div class='posts'>
+    <h5>Post # <%= @post.id %> by <%= @post.users.Name %></h5>
+    <span class='comment_like'>
+      Comments: <%= @post.CommentsCounter %>, 
```

Comment:
- [ ] Great job working on the views but I think show posts is not right since you have this error when I try to show the posts so please fix the error to let me be able to see show posts views 🙏🏻💯🔥🔥

- The error
![image](https://user-images.githubusercontent.com/58553711/216202010-f93bb817-ec85-4550-8074-bf4f54b18b92.png)

- I can't see this page in your views
![image](https://user-images.githubusercontent.com/58553711/216202104-7b69fd1e-9b23-44bf-acf1-ce63b945212d.png)